docs/index.md

```diff
@@ -1,17 +1,39 @@
-# Welcome to MkDocs
+# Welcome to Podman-Py
 
-For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+This python package is a set of bindings to use the new RESTful API in
+[libpod](https://github.com/containers/libpod).  It is currently under
```

Comment:
I think pointing at Podman rather than libpod is better here.  @mheon do you have an opinion?

```suggestion
[podman](https://github.com/containers/podman).  It is currently under
```

Reply:
Done. Should I also change the same line in the README? It's copied from there.

Reply:
@erikbgithub If you wouldn't mind doing a separate PR, that would be great.  Thanks!